pkg/kubelet/eviction/threshold_notifier_linux.go

```diff
@@ -107,13 +107,13 @@ func NewCgroupNotifier(path, attribute string, threshold int64) (CgroupNotifier,
 	}, nil
 }
 
-func (n *linuxCgroupNotifier) Start(eventCh chan<- struct{}) {
+func (n *linuxCgroupNotifier) Start(logger klog.Logger, eventCh chan<- struct{}) {
```

Comment:
the logger is passed to the `NewCgroupNotifier`. Should we just remember it there and not ask for each method?

If we want a different logger in construictor and `Start`, the better way is to use `ctx` here. Semantic will be clearer